### ISSUE TITLE
Ingress component 

### ISSUE THREAD

**Issue by fculpo** [grafana/cortex-jsonnet#373]
Hi,

The official helm chart deploys nginx replicas and allows creation of an Ingress to forward incoming traffic to the correct components (mainly distributors).

How is this handled by this project ? I don't see any reference to Ingress, nor webserver component.

**Comment by pracucci** [comment]
> As I understand it Grafana themselves run an authentication proxy layer in front of distributors in their Grafana Cloud setup, which understandably isn't included in this repo.

Correct. We use a proprietary authentication layer coupled with our infrastructure.
